components/create_team/components/team_url/index.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2018-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {bindActionCreators} from 'redux';
+import {connect} from 'react-redux';
+
+import {checkIfTeamExists, createTeam} from 'mattermost-redux/actions/teams';
+
+import TeamUrl from './team_url';
+
+function mapStateToProps(state, ownProps) {
+    return {
+        ...ownProps
+    };
+}
```

Comment:
I know we actually pass in `...ownProps` everywhere in our other components but that's actually a mistake (it already gets passed in), so in this case you can just remove the `mapStateToProps` function altogether

We actually have a PR to remove them all at #798